jujugui/static/gui/src/app/components/search-results/item/item.js

```diff
@@ -195,6 +201,57 @@ YUI.add('search-results-item', function(Y) {
     },
 
     /**
+      Closes the search results.
+
+      @method _close
+    */
+    _close: function() {
+      this.props.changeState({
+        search: null,
+        profile: null
+      });
+    },
+
+    /**
+      Deploy the entity.
+
+      @param entity {Object} The entity to deploy.
+      @param evt {Object} The click event.
+    */
+    _handleDeploy: function(entity, evt) {
+      if (entity.type === 'charm') {
```

Comment:
Great suggestion, I didn't know about that method. I've updated it, probably needs another QA/review now.